clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -849,4 +849,9 @@ def warn_missing_include_dirs : Warning<
 
 def err_drv_malformed_warning_suppression_mapping : Error<
   "failed to process suppression mapping file '%0': %1">;
+
+def warn_drv_openacc_without_cir
+    : Warning<"OpenACC directives will result in no runtime behavior, use "
```

Comment:
```suggestion
    : Warning<"OpenACC directives will result in no runtime behavior. Use "
```

Reply:
We typically don't do full-stops during diagnostics as they are not sentences.  we sometimes use semi-colons though, which is probably more fitting here.